programs/fileio.c

```diff
@@ -1463,9 +1483,11 @@ static unsigned long long FIO_decompressGzFrame(dRess_t* ress,
 
     if (strm.avail_in > 0) memmove(ress->srcBuffer, strm.next_in, strm.avail_in);
     ress->srcBufferLoaded = strm.avail_in;
-    ret = inflateEnd(&strm);
-    if (ret != Z_OK) EXM_THROW(32, "zstd: %s: inflateEnd error %d", srcFileName, ret);
-    return outFileSize;
+    if (inflateEnd(&strm) != Z_OK) {
+        DISPLAYLEVEL(1, "zstd: %s: inflateEnd error \n", srcFileName);
```

Comment:
We should probably only display this if `decodingError == 0`, since `inflateEnd()` will return failure if the stream is incomplete.